### PR TITLE
Icon improvements

### PR DESCRIPTION
**Pull request by SiriusXT**
Mainly add missing icon tips
图片

### CODE REVIEW

**Review by JYC333 — APPROVED** [review]

File: docs/frontend_api/entities_fnote.js.html [hunk 1]
```diff
@@ -42,9 +42,9 @@ <h1 class="page-title">Source: entities/fnote.js</h1>
     "code": "bx bx-code",
     "render": "bx bx-extension",
     "search": "bx bx-file-find",
-    "relationMap": "bx bx-map-alt",
+    "relationMap": "bx bxl-graphql",
```
Comment (eliandoran) [inline docs/frontend_api/entities_fnote.js.html:45]: Hmm... are we sure we want to associate the relation map icon to the one of GraphQL?

image

**Review by eliandoran — CHANGES_REQUESTED** [review]
Please also solve the merge conflict.

File: docs/frontend_api/entities_fnote.js.html (hunk 1 above)
Comment (SiriusXT) [reply docs/frontend_api/entities_fnote.js.html:45]: Previously, I didn't know that GraphQL was the name of a query language for APIs; I thought it was just an ordinary name.
I feel that using the following icons for the relation map and note map might be better:
1: 
图片
2.
图片
3.
图片

Comment (eliandoran) [reply docs/frontend_api/entities_fnote.js.html]: (1) seems wonderful.

**Review by JYC333 — APPROVED** [review]

**Review by eliandoran — APPROVED** [review]

**Pull request merged**